components/brave_rewards/resources/rewards_page/lib/locale_strings.ts

```diff
@@ -115,6 +115,16 @@ export const localeStrings = {
   onboardingTextItem1: 'Get paid for private ads you see in Brave',
   onboardingTextItem2: 'Get special benefits and discounts',
   onboardingTitle: 'Turn on Brave Rewards',
+  payoutAccountBalanceLabel: 'Your balance',
+  payoutAccountConnectedLabel: 'Connected',
+  payoutAccountDetailsTitle: '$1 wallet',
+  payoutAccountLabel: 'Payout account',
+  payoutAccountLink: 'Go to my  account',
```

Comment:
Nit: This entry has an extra space in it :-)

Reply:
Fixed